src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.*;
+
+public class Tagging {
```

Comment:
Lacking a header comment here. An example could be : Represents an adding or removing a Tag to/from a Person